nav2_mppi_controller/test/utils_test.cpp

```diff
@@ -367,8 +367,8 @@ TEST(UtilsTests, SmootherTest)
   EXPECT_NEAR(history[3].wz, 0.23, 0.02);
 
   // Check that path is smoother
-  float smoothed_val, original_val;
-  for (unsigned int i = 1; i != noisey_sequence.vx.shape(0) - 1; i++) {
+  float smoothed_val{0}, original_val{0};
+  for (unsigned int i = 0; i != noisey_sequence.vx.shape(0); i++) {
```

Comment:
Why reindex?

Reply:
Not intended - looks like a difference between humble and main branches

`humble` has:

https://github.com/ros-planning/navigation2/blob/6e35cc7417463c3f441c0d7fc596cff251cdfe4b/nav2_mppi_controller/test/utils_test.cpp#L358-L360

Reply:
Reverted in https://github.com/ros-planning/navigation2/pull/3831/commits/d7a107e3325361a894b52e22986b15af7a06af1c